tools/Common.Netcore.Dependencies.targets

```diff
@@ -2,8 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.4.0" />
-    <PackageReference Include="Microsoft.Azure.Common" Version="2.1.4" />
-    <PackageReference Include="PowerShellStandard.Library" Version="3.0.0-preview-01" />
+    <PackageReference Include="Microsoft.Azure.Common" Version="2.2.0" />
```

Comment:
Would these be the same dependencies that should exist in Common.Netcore.Dependencies.Test.targets? (file doesn't exist yet, but will in due time)

Reply:
So, I think the netcore common test should include the test libraries and PowerShell SDK.  Once we have two or three converted we can add additional libraries to common